corehq/apps/cloudcare/static/cloudcare/js/formplayer/layout/views/progress_bar.js

```diff
@@ -23,6 +23,12 @@ hqDefine("cloudcare/js/formplayer/layout/views/progress_bar", function () {
                 $(this.el).find('.progress').addClass("hide");
                 $(this.el).find('.js-loading').removeClass("hide");
             } else {
+                if (hqImport('hqwebapp/js/toggles').toggleEnabled('USE_PROMINENT_PROGRESS_BAR')) {
+                    $(this.el).find('.progress').css("height", "12px");
```

Comment:
not sure how much cost is associated with `$(this.el)`, but you could create a const for it

Reply:
that is a good idea. even if the cost is small

Reply:
added a property that is used here and in `utils` https://github.com/dimagi/commcare-hq/pull/34158/commits/3c1845c2de4649dcc7804e3463e34d779678b8f9